app/templates/static/form_handler.js

```diff
@@ -137,9 +142,17 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
                         const x = (parseFloat(target.getAttribute('data-x')) || 0) + event.dx;
                         const y = (parseFloat(target.getAttribute('data-y')) || 0) + event.dy;
 
-                        target.style.transform = `translate(${x}px, ${y}px)`;
-                        target.setAttribute('data-x', x);
-                        target.setAttribute('data-y', y);
+                        // Ensure the text stays within the bounds of the image wrapper
```

Comment:
Do we still need this?

Reply:
Thank you for the feedback, @lgaleana.

You're right. With the new restriction logic in place, the previous code for setting the transform and data attributes directly is no longer necessary. I'll remove that part to keep the code clean and focused on the new logic.

I'll make the necessary adjustments and push the changes shortly.